aueb/plugin.go

```diff
@@ -30,16 +34,35 @@ type RoomMap struct {
 
 // PLUGIN The plugin to be read by the moniteur agent.
 var PLUGIN = Plugin{}
+
+// Provides map for english-greek room name translation
 var mapping = &RoomMap{}
```

Comment:
[minor] You can use the:
```go
var(
  variable = "string"
)
```
syntax here!

Reply:
Ok whatever done.